tests/integration/customizations/s3/test_plugin.py

```diff
@@ -466,8 +527,10 @@ def test_copy_large_file_signature_v4(self):
     def test_copy_metadata(self):
         # Copy the same style of parsing as the CLI session. This is needed
         # For comparing expires timestamp.
+        # TODO: james - do we need this?  seems like we shouldn't be doing
+        # this.
         add_scalar_parsers(self.session)
```

Comment:
Ah crap, sorry about that.  I meant to remove that, it's more of an internal note for me.

I'd rather just use the real session via `create_clidriver().session`.  Given these are integration tests, that seems reasonable to use as realistic of a session as possible.

In the meantime I'll remove this.